src/test/groovy/com/github/benmanes/gradle/versions/ConstraintsSpec.groovy

```diff
@@ -131,4 +131,70 @@ final class ConstraintsSpec extends Specification {
     result.output.contains('No dependencies found.')
     result.task(':dependencyUpdates').outcome == SUCCESS
   }
+
+  def "Does not show updates for an gradle constraint added in 7.3.2"() {
+    given:
+    ExpandoMetaClass.disableGlobally()
+    def srdErrWriter = new StringWriter()
+    buildFile = testProjectDir.newFile('build.gradle.kts')
+    buildFile <<
+      """
+        plugins {
+            java
+            id("com.github.ben-manes.versions")
+        }
+
+        tasks.withType<com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask> {
+            checkConstraints = true
+        }
+      """.stripIndent()
+
+    when:
+    def result = GradleRunner.create()
+      .withGradleVersion('7.3.2')
+      .withProjectDir(testProjectDir.root)
+      .withArguments('dependencyUpdates')
+      .forwardStdError(srdErrWriter)
+      .withPluginClasspath()
+    //.forwardOutput() // Affiche les logs dans la console
+      .build()
+
+    then:
+    !result.output.contains('org.apache.logging.log4j:log4j-core [2.16.0 -> 2.17.0]')
+    srdErrWriter.toString().empty
+    result.task(':dependencyUpdates').outcome == SUCCESS
+  }
+
+  def "Does not show updates for an gradle constraint added in 6.9.2"() {
+    given:
+    ExpandoMetaClass.disableGlobally()
+    def srdErrWriter = new StringWriter()
+    buildFile = testProjectDir.newFile('build.gradle.kts')
+    buildFile <<
+      """
+        plugins {
+            java
+            id("com.github.ben-manes.versions")
+        }
+
+        tasks.withType<com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask> {
+            checkConstraints = true
+        }
+      """.stripIndent()
+
+    when:
+    def result = GradleRunner.create()
+      .withGradleVersion('6.9.2')
```

Comment:
I think you can use this sort of pattern to test the multiple versions

https://github.com/ben-manes/gradle-versions-plugin/blob/master/src/test/groovy/com/github/benmanes/gradle/versions/DifferentGradleVersionsSpec.groovy#L100

Reply:
Right, I thought about it at some point, and forgot :)

Reply:
done